dask_jobqueue/pbs.py

```diff
@@ -34,6 +34,19 @@ def pbs_format_bytes_ceil(n):
     return "%dB" % n
 
 
+def pbs_format_resource_spec(resource_spec, worker_cores, worker_memory):
```

Comment:
See above comment on the LSF templating/filtering

Reply:
Just giving it a try here. I think in the template it would look something like:
```j2
{% if resource_spec is not none -%}
#PBS -l {{ resource_spec }}
{% else -%}
#PBS -l select=1:ncpus={{ worker_cores }}:mem={{ worker_memory | pbs_format_bytes_ceil }}
{% endif -%}
```

I like that because it's clear with a single look at the template file what is done. Thoughts?

Reply:
Agreed! That's much more simple. I wonder if we should filter on `worker_cores` that ensures that it is an integer?

```
#PBS -l select=1:ncpus={{ worker_cores | round | int }}:mem={{ worker_memory | pbs_format_bytes_ceil }}
```